README.md

```
@@ -1,10 +1,90 @@
+
+1. Clone the repository:
+
+```bash
+git clone https://github.com/DevlinRocha/portfolio.git
+
+# or
+
+git clone git@github.com:DevlinRocha/portfolio.git
+```
+
+2. Change directories and install the modules:
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)